### DISCUSSION TITLE
Github Approvers nomination for the First vote Build vs. Burn

### DISCUSSION THREAD

**Discussion by Phyloiota** [iota-community/Community-Governance#52] [General]
As discussed in the community call in the IOTA Discord yesterday 04/11/21 a set of people is needed to approve the file (JSON / binary) which will be used by the nodes, the Firefly Wallet and the voting application to display and execute, and to count the first vote in the Tangle.
- The proposal for the first vote will need to follow a predefined form (based on RFC by Walter, adjusted by Alex). The texts that users will see in the Firefly wallet for Vote questions and possible answers are defined in this proposal file.
- The hornet node operators, the IOTA Firefly Nodes, and the Treasury Website will use this file to display the vote information and count the votes. To make sure that only the correct data is included in this file it will need to be proposed in the community treasury Github Repository, checked by a set of approvers, and approved to be merged into the Main branch of the Repo.
- This check, approval, and merging is a crucial security part of the vote and the community needs to decide which persons they want to ask to take these checks and the responsibility of approving the proposal.
- The community will need to decide on the number of persons that need to come to a unanimous agreement and approve it in Github. A suggested number in the call was 10 people. Vote on the number in this poll: https://discord.com/channels/397872799483428865/903229208824778793/906029145241288726
- It was discussed that people could be suggested by the community, and also that people could nominate themselves as approvers of this.
- These people would receive approver privileges in the community Treasury Github Repo and a rule would be created that the proposal can only be merged into the Main Branch and a release created if all those 10 people have checked and approved the proposal.

- Please use this discussion thread to nominate persons that you would want to take on this task for the community.
Give a upvote emoji to support suggestions

- If you want to nominate yourself to be an approver do it here in the tread and receive thumbs-ups.
- edit 12/11 - you can give upvotes to multiple candidates that you deem suitable!

**The final decision will be undertaken in our next community call on November 18th**

**Comment by dmitrdmi** [comment]
Alex Sporn

**Comment by JamesKunisch** [comment]
Adamski

**Comment by Bifonix** [comment]
Edur

**Comment by johanneskeim** [comment]
Jack Kerouac#0567

**Comment by zetec84** [comment]
Linus Naumann

**Comment by foofork** [comment]
Dylan Tullberg / foofork

**Comment by AdamEunson** [comment]
Merul Dhiman / Coodos

**Comment by kilianhln** [comment]
Dr. Muon / Dr. Electron

**Comment by antonionardella** [comment]
Dave[EF] / daverl

**Comment by Laziegoblin** [comment]
Scissors/Sissors#7808

**Comment by OMSKremer** [comment]
vrom

**Comment by danielkberlin** [comment]
ThomasQv#9150

**Comment by zarte13** [comment]
Camus#8949

**Comment by camusdefinitely** [comment]
gman214#2041

**Comment by vromvonbeyond** [comment]
MicroEngeneer / pmaxuw

**Comment by KimJongup** [comment]
KimJongUP

**Comment by J-onb** [comment]
Jonb #9519

**Comment by Mke91** [comment]
Mathieu#0042 / Chris [better than] Alice

**Comment by Tree-v2** [comment]
NWiOta

**Comment by sascha1337** [comment]


**Reply by Phyloiota** [reply]
Do you want to nominate yourself, or Merul? Merul is already nominated btw. Please adjust your suggestion. Only one candidate per nomination.

**Reply by Phyloiota** [reply]
@sascha1337 removing your comment now to not cause confusion as you have nominated 2 persons in one thread. Please resubmit if you want to nominate yourself, Merul is already nominated so people can vote for him.
Deleting to not cause confusion.

**Reply by sascha1337** [reply]
np ser

**Comment by w4Iker** [comment]
w4lker

**Comment by Sn0rra** [comment]


**Reply by dmitrdmi** [reply]
He's already nominated, pls scroll up

**Reply by Phyloiota** [reply]
@Sn0rra - removing your nomination as Edur is already nominated above to keep this on topic and not cause confusion.

**Comment by Sn0rra** [comment]
Thx Mate, this was a mistake. Doing the Vote vor the first time!

Greetings , snorra

Phyloiota ***@***.***> schrieb am Di., 9. Nov. 2021, 01:29:

**Comment by Citrullin** [comment]
David Sønstebø

**Comment by qubic9** [comment]
Sergey Ivancheglo

**Comment by Cryptec** [comment]
muXxer#1289

**Comment by coodos** [comment]
Jeoren/jvdhout#4402

**Comment by ghost** [comment]
gman214#2041

**Reply by Phyloiota** [reply]
Ben, gman is already suggested, so i delete / hide your nomination here, thanks

**Comment by Apahatschi** [comment]
NO8ODY#9999

**Comment by PuckStar** [comment]
Kevin / MudKevin#7889

**Comment by NiklasWillecke** [comment]
Werner#2923

**Comment by emeliebob** [comment]
luka#2523

**Comment by breakle** [comment]
Bas / Bas#7510

**Comment by Hagi09** [comment]
Microhash (Lukas Tassanyi)

**Comment by Hagi09** [comment]
Werner (VOTE BUIDL!)

**Reply by basvansambeek** [reply]
Werner is already mentioned by Niklas

**Reply by Phyloiota** [reply]
i am hidding this post Hagi, as Werner is already nominated, so it does not lead to confusion.

**Comment by Hagi09** [comment]
Chris Mueller

**Comment by NiklasWillecke** [comment]
Kappy84#0119

**Comment by ghost** [comment]
Ben Royce

**Comment by ghost** [comment]
Alex Renz

**Comment by ghost** [comment]
IOTerry

**Comment by specmanic** [comment]
Rajiv#5322 ... ah btw happy birthday Rajiv 🎉

**Comment by ghost** [comment]
Daniel De Michele (Carpincho Dem)

**Comment by ghost** [comment]
Gabriela Jara

**Comment by Phyloiota** [comment]
These are the final results:
![image](https://user-images.githubusercontent.com/77154511/142516188-67483926-a0ef-4efc-aaef-8d6d80587ca7.png)
The community has decided in the call on 18/11 to choose the top 13 of that list and require the approval of 10 out of these 13, this means Alex Sporn is number 13. 
Thanks to everyone that made suggestions here. The conversation is locked and the nominated candidates will be informed once it's time to proceed.

**Comment by Phyloiota** [comment]
Our selected approvers are invited to check the correct implementation of the texts for the vote in this event:

https://github.com/iotaledger/participation-events/blob/master/events/vote/governance_01.json

**Comment by viossat** [comment]
LGTM

**Comment by OMSKremer** [comment]
LGTM. I confirm that the governance_01.json (github commit e659c36530983f0808c5f62b5b5d56a8893cfb31) is technically and contentwise correct. (Me, Discord: Edur#0001)

**Comment by shufps** [comment]
LGTM :blush:

**Comment by alexsporn** [comment]
LGTM

**Comment by coodos** [comment]
LGTM
`governance_01.json` at commit `e659c36530983f0808c5f62b5b5d56a8893cfb31` 
(merul#0001 on discord)

**Comment by gmanredmond** [comment]
LGTM. I confirm that the governance_01.json (github commit e659c36530983f0808c5f62b5b5d56a8893cfb31) is technically and contentwise correct. (Me, Discord: Edur#0001)

**Reply by OMSKremer** [reply]
That's not edur! ;P
